test/ui-test/testSuites/suite_onboarding/tst_statusSignUp/test.feature

```diff
@@ -142,7 +142,7 @@ Feature: Status Desktop Sign Up
   @merge
   Scenario: After Signing up the Profile state should be online
     Given A first time user lands on the status desktop and generates new key
-    When user signs up with username "tester123" and password "TesTEr16843/!@00"
+    And the user signs up with username "tester123" and password "TesTEr16843/!@00"
     Then the user is online
 
   @merge
```

Comment:
Suggestion to break this into 2 separate scenarios:

Scenario: User signs up with a profile image
Given A first time user lands on the status desktop and generates new key
When the user signs up with profileImage doggo.jpeg, username tester123 and password TesTEr16843/!@00
Then my profile modal has the updated profile image
And the profile setting has the updated profile image

Given the user restarts the app
And a screenshot of the profileImage is taken
When the user logs in with password TesTEr16843/!@00
Then the profile navigation bar has the updated profile image